CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ All notable, unreleased changes to this project will be documented in this file.
 - Added caching of GraphQL documents for common queries to improve performance - #14843 by @patrys
 - Added `VOUCHER_CODES_CREATED` and `VOUCHER_CODES_DELETED` webhooks events. - #14652 by @SzymJ
 - Fixed validation for streetAddress1 or streetAddress2 are too long - #13973 by sonbui00
+- Remove `prefetched_for_webhook` to legacy payload generators - #15369 by @AjmalPonneth
```

Comment:
Since we released 3.19 this entry should be moved to 3.20 now (you need to rebase with main).